write-fonts/src/tables/glyf.rs

```diff
@@ -295,4 +344,81 @@ mod tests {
         assert_eq!(points[4].y, 256);
         assert!(points[4].on_curve);
     }
+
+    #[test]
+    fn compile_repeatable_flags() {
+        let mut path = BezPath::new();
+        path.move_to((20., -100.));
+        path.line_to((80., -20.));
+        path.line_to((50., -69.));
+        path.line_to((25., -90.));
```

Comment:
Optional, BezPath::from_svg("M20,-100 L80,-20 etc") might be a more concise way to write this sort of test input?